public/index.php

```diff
@@ -14,8 +14,14 @@
     unset($path);
 }
 
-// Setup autoloading
-require 'init_autoloader.php';
+// Composer autoloading
+if (!file_exists('vendor/autoload.php')) {
```

Comment:
This `if` block can actually be skipped, as it causes a stat call. Not sure if it would make the FW actually harder to use tho :-(